docker/bookkeeper/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update \
 &&  mkdir -p /opt \
 &&  cd /opt
 
-ENV BOOKKEEPER_VERSION=4.5.0 \
```

Comment:
Would this mean we are running 4.6.1 version of server against 4.5.0 client?
IF this is the case, we should add a comment in the `gradle.properties` which says this. Otherwise this may fail. All our integration tests will run 4.5.0 server and this will be different than the actual deployment and system tests.
This should be documented.

Reply:
done. documented in `gradle.properties`.